src/RawHtmlFieldInput.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {changeBlockValue} from './actions';
+import {
+  getFieldName,
+  isStatic,
+  shouldRunInnerScripts
+} from './processing/utils';
+
+
+const MutationObserver = window.MutationObserver
+                      || window.WebKitMutationObserver
+                      || window.MozMutationObserver;
```

Comment:
MDN says the prefixes are from browser versions from 6 years ago, so they should be removed.